packages/files-ui/cypress/tests/sharing-explainer-spec.ts

```diff
@@ -32,6 +33,30 @@ describe("Sharing Explainer", () => {
       })
     })
 
+    it("User can dismiss the sharing explainer with the cross", () => {
+      // intercept and stub the response to ensure the explainer is displayed
+      cy.intercept("GET", "**/user/store", {
+        body: { [sharingExplainerKey]: "false" }
+      })
+
+      cy.web3Login()
+      navigationMenu.sharedNavButton().click()
+      sharingExplainerModal.body().should("be.visible")
+
+      // set up a spy for the POST response
+      cy.intercept("POST", "**/user/store").as("storePost").then(() => {
+
+        // dismiss the sharing explainer
+        sharingExplainerModal.closeButton().click()
```

Comment:
```suggestion
        sharingExplainerModal.closeButton().safeClick()
```

I tend to use `safeClick` for modal closures to remove any chance of "detached from DOM" issues or clicking before an event listener has been registered etc.

Reply:
The safeclick is really meant for buttons, as we're checking for the `enabled` state, which is not available on this cross image. I went back to the click. We can figure it out if we ever have an issue. Safeclick remains kind of a hack that I'd rather use only when needed.

Reply:
Ah ok 👍, just by looking at the test I thought it was a button. I guess to the user it is, at least functions as one, but if the element type is not actually `button` then yeah no need to use `safeClick`